src/main/java/mat/server/util/MeasureUtility.java

```diff
@@ -98,24 +96,38 @@ public static String cleanString(String originalString) {
         return cleanedString;
     }
 
+    /**
+     * Update CQL version.
+     *
+     * @param processor the processor
+     * @param version
+     */
+    public static void updateCQLVersion(XmlProcessor processor, String version) throws XPathExpressionException {
+        Node versionNode = processor.findNode(processor.getOriginalDoc(), "//cqlLookUp/version");
+        if (versionNode != null) {
+            versionNode.setTextContent(version);
+        }

```

Comment:
Just a suggestion. Could refactor the search and replace logic into its own method that takes processor, xpath, and replacement text to reduce code duplication.

Reply:
Agree, makes sene. I could not do that due to error handling differences. But now we can do that.